internal/rpc/health/types.go

```diff
@@ -0,0 +1,9 @@
+package health
+
+import "github.com/NethermindEth/juno/pkg/types"
+
+type Status struct {
+	Status        string            `json:"status"`
+	Message       string            `json:"message"`
+	SyncingStatus *types.SyncStatus `json:"syncingStatus"`
+}
```

Comment:
This struct alone doesn't warrant a separate file. Also, the file name should be a little more specific about what is contained within. `types.go` is too generic a name.